docs/source/principles.md

```diff
@@ -47,22 +47,23 @@ TREs holding public data should build and maintain the trust of data subjects an
 
 ### Rationale
 
-Maintaining trust in TREs to hold and use data is essential to prevent backlash or resistance to the use of data.
-This can include maintaining the trust of members of the public who are impacted by research, as well as the trust of commercial data providers.
+To ensure continued public support for the use of data and to alleviate possible concerns, it is vital to maintain public trust in the ways TREs hold and use data.
+This can include maintaining the trust of members of the public whose data is held, those who are impacted by research, as well as the trust of commercial data providers
 
-In the case of public sector data, public engagement work has indicated there is widespread support for the use of regulated and ethical TREs working for the public benefit.
+In the case of public sector data, public engagement work has indicated there is support for the use of regulated and ethical TREs working for the public benefit as long as conditions are met surrounding security and transparency.
 Consulting impacted parties, including the public, can help ensure TREs are being used for positive, impactful and agreed-upon purposes.
 Being transparent about the data held and the projects or organisations who access the data can also help maintain trust.
 
 ### Implications
 
 - Being as transparent as possible is key to building trust.
-  TREs holding public data should practice transparency, for instance making adherence to any accreditation body, or framework (for instance the Five Safes) publicly available information, and making information on the projects or organisations which access their data available to impacted parties.
-- A strong public engagement programme takes time and resource.
-  TREs holding public data should consider allocating specific staffing to public engagement activities.
+  TREs holding public data should practice transparency, for instance making adherence to any accreditation body, or framework (for instance the Five Safes) publicly available information, and making information on the projects or organisations which access their data available.
+  This should be written in an accessible way.
+- In addition to transparency, actively involving members of the public in oversight of TREs and their processes is important for accountability.
+  Public involvement and engagement takes time and resource. TREs holding public data should consider allocating specific staffing and budget to public engagement activities.
 - TREs open to consultation by impacted parties should be auditable, and impacted parties should be part of any decision making processes.
```

Comment:
My only thought is on the bit that hasn't changed :-) @Katie-rds What does "TREs open to consultation by impacted parties should be auditable" mean? Does it mean TREs should be auditable by anyone impacted by a TRE? Or does it mean if TREs choose to make themselve open to consultation by the public then they must be auditable?

Everything else looks fine to me

Reply:
That's the bit I've queried in the detail. I didn't write it originally so didn't want to remove it without checking. Think @machintim wrote it?

Reply:
On one hand it's already in the spec so it doesn't matter if it remains even if it's unclear. On the other hand it'd be nice to ensure the wording is clear before our 1.0.0 release. I'm leaning towards removing it, but happy to go with whatever everyone else prefers.

Reply:
I'm happy to remove it and not have something unclear in and if it's important it can be added after 1.0.0 release?

Reply:
```suggestion
```

Reply:
It's harking back to this: 
```
Consulting impacted parties, including the public, can help ensure TREs are being used for positive, impactful and agreed-upon purposes
```

I think lack of clarity is that the current wording implies that it's the impacted parties who consult the TREs, when it should be the TREs who consult the impacted parties.

Maybe something like

```suggestion
- Where TREs have consulted impacted parties, they should be auditable by those parties, and consult them as part of any decision making processes.
```
is clearer (but still a bit clunky, and just toss it if I'm muddying the waters)?